docs/installation/mac.md

```diff
@@ -255,7 +274,7 @@ and running
 
 <div class="admonition note">
 
-Note: See https://github.com/GMLC-TDC/HELICS/issues/763 if your installation doesn't point the dylib to the correct location.
+Note: See [https://github.com/GMLC-TDC/HELICS/issues/763](https://github.com/GMLC-TDC/HELICS/issues/763) if your installation doesn't point the dylib to the correct location.
```

Comment:
Isn't this something we can fix easily?

Reply:
It could be easy to fix -- we should probably work on that issue soon.